pkgs/development/python-modules/gym/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib
+, buildPythonPackage, fetchPypi
+, numpy, requests, six, pyglet, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "gym";
+  version = "0.9.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0llbhn3zdlsz2crd5grd1yygg8zp2shsclc24iqix5gw5f65clx5";
+  };
+
+  propagatedBuildInputs = [
+    numpy requests six pyglet scipy
+  ];
+
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "The OpenAI Gym: A toolkit for developing and comparing your reinforcement learning agents.";
```

Comment:
https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

Reply:
If it's about `meta.platforms`, it's already set by `buildPythonPackage` :)